tests/testConstantSettings.php

```diff
@@ -18,53 +18,21 @@ class TestConstantSettings extends TestCase {
 
 	use domDocumentHelpers;
 
-	/**
-	 * Test string to use.
-	 */
-	const FILTER_TEST_STRING = '__filter_test__';
-
 	/**
 	 * Default constant setting prefix.
 	 *
 	 * @see WP_Auth0_Options_Generic::get_constant_name()
 	 */
-	const DEFAULT_CONSTANT_PREFIX = 'AUTH0_ENV_';
-
-	/**
-	 * Notice text for a field with a constant value set.
-	 *
-	 * @see WP_Auth0_Admin_Generic::render_const_notice()
-	 */
-	const CONSTANT_NOTICE_TEXT = 'Value is set in the constant';
-
-	/**
-	 * Test that setting a constant will store the constant key.
-	 *
-	 * @runInSeparateProcess
-	 */
-	public function testThatConstructorStoresConstants() {
```

Comment:
Combined with `testThatConstantValuesAreUsed` to run all "is it set" assertions at once